applications/ws-affiliation-rnsr/Dockerfile

```
@@ -1,29 +1,23 @@
 # syntax=docker/dockerfile:1.2
 FROM inistcnrs/lodex-workers-python:4.0.11 as build1
-RUN --mount=type=secret,id=webdav_login \
-    --mount=type=secret,id=webdav_password \
-    --mount=type=secret,id=webdav_url \
+RUN --mount=type=secret,id=webdav_login cat /secrets/login \
+    --mount=type=secret,id=webdav_password cat/secrets/password \
```

```suggestion
    --mount=type=secret,id=webdav_password
```

L'usage de `cat` était erroné sur cette ligne et la suivante (en plus d'être inutile ici): il n'y avait pas d'espace entre la commande `cat` et le chemin du fichier à afficher `/secrets/password`.